README.rst

```diff
@@ -0,0 +1,254 @@
+
+This example uses the `AliPhysics <https://github.com/alisw/AliPhysics>`_
+analysis framework with the following source code files:
+
+- `env.sh <env.sh>`_ - ALICE LEGO train system configuration
+- `fix-env.sh <fix-env.sh>`_ - to fix the configuration environment variables
+- `runEx01.C <runEx01.C>`_ - the main script to be run
+- `plot.C <plot.C>`_ - script to extract data and plot the figures from the resulting root file
+- `AliAnalysisTaskEx01.cxx <AliAnalysisTaskEx01.cxx>`_ - the example script analysing the Pt spectrum
+- `AliAnalysisTaskEx01.h <AliAnalysisTaskEx01.h>`_ - root library for customization
+
+3. Compute environment
+----------------------
+
+This example uses `AliPhysics <https://github.com/alisw/AliPhysics>`_ analysis
+framework. It has been containerised as `reana-env-aliphysics
+<https://github.com/reanahub/reana-env-aliphysics>`_ environment. You can fetch
+some wanted AliPhysics version from Docker Hub:
+
+.. code-block:: console
+
+   $ docker pull reanahub/reana-env-aliphysics:vAN-20180614-1
+
+We shall use the ``vAN-20180614-1`` version for the present example.
+
+Note that if you would like to build a different AliPhysics version on your own,
+you can follow `reana-env-aliphysics
+<https://github.com/reanahub/reana-env-aliphysics>`_ procedures and set
+``ALIPHYSICS_VERSION`` environment variable appropriately:
+
+.. code-block:: console
+
+   $ cd src/reana-env-aliphysics
+   $ export ALIPHYSICS_VERSION=vAN-20180521-1
+   $ make build
+
+
+4. Analysis workflow
+--------------------
+This analysis example consists of a script to run the task using aliroot, and then plots the results.
+
+The computational workflow is essentially sequential in nature. We can
+use the REANA serial workflow engine and represent the analysis workflow as
+follows:
+
+.. code-block:: console
+
+              START
+               |
+               |
+               V
+   +-----------------------------------------+
+   | (0) use data or download ESD input file |
+   |                                         |
+   |    $ wget http://opendata.cern.ch/...   |
+   +-----------------------------------------+
+               |
+               | ALIESD.root
+               V
+   +-----------------------------------------+
+   | (1) perform LEGO train test run         |
+   |                                         |
+   |    $ root -b -q './runEx01.C'           |
+   +-----------------------------------------+
+               |
+               | run.logs
+               | AnalysisResults.root
+               |
+               V
+   +----------------------------------------+
+   | (2) plot sample histogram              |
+   |                                        |
+   |    $ root -b -q plot.C                 |
+   +----------------------------------------+
+               |
+               | results/plot_pt.pdf
+               | results/plot_eta.pdf
+               |
+               V
+              STOP
```

Comment:
I get errors on step 1:

```
2019-07-24 14:10:55 (14.2 MB/s) - 'AliESDs.root' saved [14283265]


job id: bcb14771-d2e2-4c1d-a587-af3a8851506f
bash: fix-env.sh: No such file or directory
```

Perhaps it needs `./fix-env.sh` i.e. with the leading path?

Does the example works for you as it is pushed in the PR?

Reply:
Added the leading path `./`